mobile/src/components/course/components/edit-course/edit-course.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { FC } from 'react';
+
+import { AppScreenName, DataStatus } from '~/common/enums/enums';
+import { CourseUpdateCategoryRequestDto } from '~/common/types/types';
+import { BackButton, Spinner, View } from '~/components/common/common';
+import { EditCategoryDropdown } from '~/components/course/components/components';
+import {
+  useAppDispatch,
+  useAppNavigate,
+  useAppSelector,
+  useEffect,
+} from '~/hooks/hooks';
+import { categoryActions, coursesActions } from '~/store/actions';
+
+import { styles } from './styles';
+
+const EditCourse: FC = () => {
```

Comment:
use EditCourseCategory component and rename it to EditCourse, instead of navigation use navigation.goBack for back button